app/bundles/EmailBundle/EventListener/EmailSubscriber.php

```diff
@@ -96,6 +103,37 @@ public function onEmailPostSave(Events\EmailEvent $event)
         }
     }
 
+    /**
+     * Add preheader text to email body.
+     */
+    public function onEmailPostSavePreheaderText(Events\EmailEvent $event)
```

Comment:
I am not fan the change HTML code of email on email save.
I think we could inject code on build email 

@escopecz can you please review it from Acquia side, what do you think?

Reply:
Yes, I'd do it on before send.

Reply:
Changed